### PR TITLE
[#417] fixing cluster3nodes benchmark

### PR DESCRIPTION
**Pull request by CodiePP**
make the benchmark work again.
just minor changes were needed.

### CODE REVIEW

File: benchmarking/cluster3nodes/configuration/log-config-explorer.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 NetworkName: mainnet
 RequiresNetworkMagic: RequiresNoMagic
+GenesisHash: 5fbffc20a6ee41093a0f6a44e6110b66a69c7cc591c714075a6c537c360e32b6
```
Comment (Jimbo4350) [inline benchmarking/cluster3nodes/configuration/log-config-explorer.yaml]: `GenesisHash` shouldn't be required in the `.yaml` files anymore.

Comment (CodiePP) [reply benchmarking/cluster3nodes/configuration/log-config-explorer.yaml]: the explorer still depends on this.

**Pull request merged**